### ISSUE TITLE
Ensure unit tests run cleanly for encodings/fastpfor/js/

### ISSUE THREAD

**Issue by springmeyer** [maplibre/maplibre-tile-spec#38]
Currently the unit tests appear to pass but still result in a failure due to a minor issue with nyc & Ava usage:

>   ✘ No tests found in src/tests/benchmarks/fastpfor.ts, make sure to import "ava" at the top of your test file

**Comment by springmeyer** [comment]
Looks like there is another problem coming up on CI:

```
> fastpfor-ts@1.0.0 build:module
> tsc -p tsconfig.module.json

src/index.ts:12:28 - error TS2307: Cannot find module './BitPacking' or its corresponding type declarations.

12 import { fastunpack } from './BitPacking';
                              ~~~~~~~~~~~~~~


Found 1 error in src/index.ts:12

src/index.ts:12:28 - error TS[23](https://github.com/mactrem/cov-tiles/actions/runs/9228505345/job/25392651682#step:5:24)07: Cannot find module './BitPacking' or its corresponding type declarations.

12 import { fastunpack } from './BitPacking';
                              ~~~~~~~~~~~~~~
```

https://github.com/mactrem/cov-tiles/actions/runs/9228505345/job/25392651682#step:5:14

**Comment by springmeyer** [comment]
Fixed by #50
